Plugins/GRPCSwiftPlugin/plugin.swift

```diff
@@ -85,6 +85,9 @@ struct GRPCSwiftPlugin {
 
     /// A list of invocations of `protoc` with the `GRPCSwiftPlugin`.
     var invocations: [Invocation]
+
+    /// Whether this configuration is for `grpc-swift` v2.
+    var _V2: Bool?
```

Comment:
Sorry, I didn't missed this the first time around but this should be part of the `Invocation` configuration `struct` above